<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>panWidget</class>
 <widget class="QWidget" name="panWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1300</width>
    <height>500</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>1200</width>
    <height>300</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" stretch="1,6">
   <item>
    <widget class="QLabel" name="lb_name">
     <property name="font">
      <font>
       <family>Georgia</family>
       <pointsize>24</pointsize>
      </font>
     </property>
     <property name="text">
      <string>_NAME</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout" stretch="0">
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_3" stretch="3,1">
       <item>
        <widget class="QScrollArea" name="scrollArea">
         <property name="widgetResizable">
          <bool>true</bool>
         </property>
         <widget class="QWidget" name="scrollAreaWidgetContents">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>1276</width>
            <height>263</height>
           </rect>
          </property>
         </widget>
        </widget>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QLabel" name="lb_commentary">
           <property name="font">
            <font>
             <family>Georgia</family>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Хотите поделиться своим мнением? - оставьте коментарий!</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPlainTextEdit" name="pte_Commentary">
           <property name="font">
            <font>
             <family>Georgia</family>
             <pointsize>12</pointsize>
            </font>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pb_submitCommentary">
           <property name="font">
            <font>
             <family>Georgia</family>
             <pointsize>14</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Отправить</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
